scribble-lib/scribble/core.rkt

```diff
@@ -162,6 +162,36 @@
 
 ;; ----------------------------------------
 
+(define-struct link-render-style (mode)
+  #:constructor-name link-render-style
+  #:property
+  prop:serializable
+  (make-serialize-info
+   (lambda (s)
+     (vector (link-render-style-mode s)))
+   #'deserialize-link-render-style
+   #f
+   (or (current-load-relative-directory) (current-directory))))
+
+(provide deserialize-link-render-style)
```

Comment:
Why not put this provide with the others?

Reply:
The `provide`s in this file are grouped more with the implementations, and the `deserialize-....` exports are consistently (I think) grouped with the deserializers.

Really, this file should be split into several, and the deserializers should be in submodules. But I didn't tackle that refactoring here.